packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -75,12 +89,21 @@ export class Snackbar implements ComponentInterface {
     this.snackbarElement.removeEventListener('MDCSnackbar:closing', (e) =>
       this.handleSnackbarHide(e)
     );
+    this.snackbarElement.removeEventListener('mouseenter', () => {
+      if (this.timeout) clearTimeout(this.timeout);
+    });
+    this.snackbarElement.removeEventListener('mouseleave', () => {
+      this.configureTimeout();
+    });
   }
 
   private configureTimeout() {
     this.snackbarInstance.timeoutMs = -1;
     if (this.inoTimeout >= 0) {
-      setTimeout(() => this.snackbarInstance.close(), this.inoTimeout);
+      this.timeout = setTimeout(
+        () => this.snackbarInstance.close(),
+        this.inoTimeout
+      );
```

Comment:
```suggestion
      this.timeout = setTimeout(this.snackbarInstance.close, this.inoTimeout);
```

Reply:
I think this simplification ist not possible. The MDC close() method is not implemented as an arrow function, hence there would be an internal error.